ods_ci/tests/Resources/Page/OCPLogin/OCPLogin.robot

```diff
@@ -15,3 +15,9 @@ Login To OCP Using API
   [Arguments]    ${username}      ${password}
   ${rc}    ${out}=    Run And Return Rc And Output    oc login $(oc whoami --show-server) -u ${username} -p ${password}  #robocop:disable
   Should Be Equal As Integers    ${rc}    ${0}
+
+Login To OCP Using API And Kubeconfig
```

Comment:
It's possible this is not needed. Can you look at this example? https://github.com/red-hat-data-services/ods-ci/blob/1ce419d3d1dee0f4954daf9f6e5d59bf5c2d49ca/ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/436__test-run-data-science-pipelines-operator-e2e-tests.robot#L31

Reply:
Thanks for the tip.
Makes me think, what is a meaning of `kconfig`, which is available in ods-ci in the job environment, if `.kube/config` holds the current token?

Reply:
Implemented, feel free to review again